tasmota/xdrv_02_mqtt.ino

```diff
@@ -225,7 +231,14 @@ void MqttDisconnect(void) {
 }
 
 void MqttSubscribeLib(const char *topic) {
-  MqttClient.subscribe(topic);
+  #if defined(USE_MQTT_AZURE_IOT)
+    // Azure IoT Hub currently does not support custom topics: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support
+    String realTopicString = "devices/" + String(SettingsText(SET_MQTT_CLIENT));
+    realTopicString += "/messages/devicebound/#";
+    MqttClient.subscribe(realTopicString.c_str());
+  #else //USE_MQTT_AZURE_IOT
```

Comment:
I'm ok with the code, however identation does not follow the Tasmota convention. The ifdefs start at the beginning of the line, and the code is identated as usual.

Reply:
Agree. I will fix this once merged.